# Request type Constants
# API definition (two methods for now)
uint8 GET_STATE = 0     # Get state
uint8 SET_STATE = 1     # Set state

# Possible state requests
uint8 NO_REQUEST=0              # No request
uint8 REQUEST_CHARGING = 1      # Request charging
uint8 REQUEST_OPERATION= 2      # Request operation

uint8 method_id             # Method id. Values defined above API
uint8 requested_mode        #
---
# Available states
uint8 IDLE_CHARGING=0
uint8 OPERATING=1
uint8 RETURNING_TO_STATION=2
uint8 ERROR=4

uint8 state                 # Variable holding the current state. Values defined above.
float32 battery_lvl         # Percent of battery